src/components/Forms/GroupForm/GroupForm.js

```diff
@@ -1,39 +1,41 @@
-import React, { Component } from "react";
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
 import { connect } from "react-redux";
 
 //import dispatch actions for create/join group
 import { createGroup, joinGroup } from "../../../actions/groupActions"
 
 import "./GroupForm.scss";
 
-class GroupForm extends Component {
+class GroupForm extends React.Component {
   constructor(props) {
     super(props)
-    console.log('props in constructor',props)
+    console.log('props in constructor', props)
     this.state = {
       "errorMessage": "",
       "input": ""
     }
   }
   validate = val => {
     if (!val) {
-      this.setState({"errorMessage": "You didn't enter anything!"})
+      this.setState({ "errorMessage": "You didn't enter anything!" })
     } else {
-      let { action, getUserData} = this.props
+      let { action, getUserData } = this.props
       console.log('dispatch action:', action)
```

Comment:
Clean up all the console.log and unused dirty codes
PD: please read the assigned issue description what is needed to be done